en/docs/tutorials/integrating-with-solace.md

```diff
@@ -0,0 +1,144 @@
+        https_endpoint = <HTTPS Endpoint of the Solace broker environment>
+        show_as_token_endpoint_url = false
+
+        [apim.gateway.environment.properties]
+        Organization = <Password of the Solace broker account>
+        DevAccountName = <Password of the Solace broker account>
+    ```
+
+5. The Solace broker gateway is now available in the API Publisher.
+    <a href="{{base_path}}/assets/img/tutorials/solace/view-solace-broker.png"><img src="{{base_path}}/assets/img/tutorials/solace/view-solace-broker.png" alt="View Solace Broker" name="View Solace Broker"></a>
+
+## Creating and deploying an API
+
+You can use the Async API specification generated from the Solace PubSub+ Hub to create APIs in WSO2 API Manager.
+
+### Step 1 - Importing the Async API from Solace
+
+Before you begin: Make sure that you have integrated the Solace event broker with WSO2 API Manager as shown in the above section.
+
+1. Login to the API Publisher via **https://&lt;hostname>:9443/publisher**
+2. Click **CREATE API** and then click **Import AsyncAPI Definition**
+    <a href="{{base_path}}/assets/img/tutorials/solace/import-asyncapi.png"><img src="{{base_path}}/assets/img/tutorials/solace/import-asyncapi.png" alt="Import AsyncAPI Definition" name="Import AsyncAPI Definition"></a>
+
+3. The following two options to import the AsynAPI definition appear.
+    1. AsyncAPI URL - If you select this option, you need to provide a URL of a Solace API Specification.
+    2. AsyncAPI File - If you select this option, click Browse File to Upload and upload a file, which contains a Solace AsyncAPI definition.
+    Select your preferred method and click **NEXT**.
+
+4. Edit the Solace Async API information and click **Create**.
+
+!!! note
+    The Async API definition of the Solace API will contain the specific API definition, with the supported protocols, such as HTTP, MQTT, AMQP, that the API topics have to use. You do not need to provide the protocol information separately
+
+### Step 2 - Configuring Topics
+
+1. Click **Topics** to navigate to the topics page.
+    <a href="{{base_path}}/assets/img/tutorials/solace/topics.png"><img src="{{base_path}}/assets/img/tutorials/solace/topics.png" alt="navigate to Topics" name="navigate to Topics"></a>
+
+The topics will be created automatically from the Solace AsyncAPI definition provided in the previous step.
+
+### Step 3 - Viewing the Async API definition
+
+1. Go to **API Configurations** and click **AsyncAPI Definition** to see the definition of the API you created in the previous step. 
+    <a href="{{base_path}}/assets/img/tutorials/solace/asyncapi-definition.png"><img src="{{base_path}}/assets/img/tutorials/solace/asyncapi-definition.png" alt="AsyncAPI Definition" name="AsyncAPI Definition"></a>
+
+    !!! note
+        The Solace API specification cannot be edited from the API manager Publisher portal. But the specification can be downloaded and changed externally to make the required changes. 
+
```

Comment:
Shall we add this part saying at the moment we only support one subscription policy option?

Suggestion: **_Warning_ - The current version of WSO2 API Manager only supports AsyncUnlimited Protocol as the API level subscription policy for Solace APIs. Once a subscription is made, that policy will be added automatically.**